index.html

```diff
@@ -1,209 +1,123 @@
-<!DOCTYPE html>
```

Comment:
don't remove the doctype please

Reply:
I didnt. Ive been going all day, so again, my phone. WTF.